src/plugins/index_pattern_management/public/components/create_index_pattern_wizard/lib/get_indices.ts

```diff
@@ -199,7 +199,14 @@ export async function getIndices({
     pattern,
     showAllIndices,
     dataSourceId,
-  }).catch(() => []);
+  }).catch((error) => {
+    // swallow the errors to be backwards compatible with non-data-source use case
```

Comment:
there is also line 103 could use the same :P

Reply:
line 103 maps to `getIndicesViaSearch` which is specifically for CCS use case as a step 2 request. But for CCS it also requires step 1 querying against local cluster. Even we added the same logic to step 2, the code won't be able to reach there, because the error will be thrown at step 1 already. Let's leave as it is.

Reply:
My point was the call through search client in getIndicesViaSearch itself could also have error. And it's a different route from the resolve api. But it's not blocking this PR though.